src/BaGet/Migrations/Sqlite/20180907104341_Initial.Designer.cs

```diff
@@ -9,7 +9,7 @@
 namespace BaGet.Migrations.Sqlite
 {
     [DbContext(typeof(SqliteContext))]
-    [Migration("20180804082808_Initial")]
+    [Migration("20180907104341_Initial")]

```

Comment:
Renaming the migration seems like a bad idea, no?

@loic-sharma

Reply:
Good catch! This should be rolled back

Reply:
sorry, u r right, don't understand how it is happen (

Reply:
I changed it back, but do not think it is right... at first: it is auto-generated file, 2nd: I see you(or scripts) made changes(add fields) to it rather than generate a new migration - it is not a right way to work with migrations